pages.zh/common/chown.md

```diff
@@ -0,0 +1,24 @@
+# chown
+
+> 修改用户和用户组对文件或目录的所有权。
+> 更多信息：<https://www.gnu.org/software/coreutils/chown>.
+
+- 修改文件或目录的所有者：
+
+`chown {{用户}}} {{文件或目录路径}}`
```

Comment:
Refer to [common-arguments.md](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md).

```suggestion
`chown {{用户}}} {{路径/到/文件或目录}}`
```